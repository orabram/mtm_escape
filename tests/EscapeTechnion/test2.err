Error: The parameter sent to the function is invalid
Error: The email already exists in the system
Error: The parameter sent to the function is invalid
Error: The room ID already exists in the faculty
Error: The email entered does not belong to any company in the system
Error: The parameter sent to the function is invalid
Error: The parameter sent to the function is invalid
Error: The parameter sent to the function is invalid
Error: The parameter sent to the function is invalid
Error: The parameter sent to the function is invalid
Error: The parameter sent to the function is invalid
Error: The parameter sent to the function is invalid
Error: The parameter sent to the function is invalid
Error: The parameter sent to the function is invalid
Error: The email already exists in the system
Error: The parameter sent to the function is invalid
Error: The parameter sent to the function is invalid
Error: The email already exists in the system
Error: The parameter sent to the function is invalid
Error: The parameter sent to the function is invalid
Error: The email entered does not belong to any client in the system
Error: The room ID does not belong to any room in the faculty
Error: The client already has an order made for this time
Error: The client already has an order made for this time
Error: The room is not available in the given time
Error: The room is not available in the given time
Error: The room is not available in the given time
Error: The company or room cannot be removed because a reservation currently exists
Error: The company or room cannot be removed because a reservation currently exists
Error: The company or room cannot be removed because a reservation currently exists
Error: The client already has an order made for this time
Error: The room is not available in the given time
Error: The room is not available in the given time
Error: The room ID does not belong to any room in the faculty
Error: The email entered does not belong to any client in the system
