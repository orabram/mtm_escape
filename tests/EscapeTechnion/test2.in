				#NANANANANANA BATMAN! company add you

company add company1@gmail 0
company add company2@gmail 2
#bad
company add company2@gmail 18
company add company2@gmail 8
company add company3gmail 8
#good
company add company3@gmail 0
company add company4@gmail 8
company add company5@gmail 16
company add company6@gmail 6
company add company7@gmail 1
#zzzzzzzzz

	room	 add	 company1@gmail 	1 		100 	4 	00-24	 9         
#bad
room add company3@gmail 1 40 4 04-05 10
room add compi@ 1 40 5 04-05 10
room add compi 1 40 5 04-05 10
room add company@gmail 1 41 5 04-05 10
room add company@gmail 1 40 0 04-05 10
room add company@gmail 1 40 4 05-05 10
room add company@gmail 1 40 4 05-25 10
room add company@gmail 1 40 4 05-25 11
room add company@gmail 1 40 4 05-25 0
room add company@gmail 1 40 4 05-25 -1
#good
room add company1@gmail 2 80 2 05-15 4
room add company2@gmail 1 4 3 08-19 1
room add company2@gmail 4 20 4 02-22 5
room add company3@gmail 5 24 4 02-22 7
room add company4@gmail 2 24 4 06-18 8
room add company5@gmail 8 60 2 08-09 2
room add company6@gmail 11 120 7 18-24 9
room add company6@gmail 14 200 2 15-23 1
room add company7@gmail 12 12 2 12-22 6

#bad
escaper add company3@gmail 0 0
escaper add company3@gmail 0 2
escaper add company3@gmail 18 1
escaper add 1 0 1
#good
escaper add 1@ 0 1
escaper add 2@ 1 4
escaper add 3@ 0 1
escaper add 4@ 1 5
escaper add 5@ 8 1
escaper add 6@ 16 7
#bad
company add 1@ 1

#good
escaper order 1@ 0 2 0-10 4
#bad
escaper order 0@ 0 2 0-10 0
escaper order 0@ 18 2 0-10 4  
escaper order 0@ 0 123 0-10 4
escaper order 1@ 0 123 0-10 4
escaper order 1@ 0 2 0-10 4
escaper order 1@ 0 5 0-10 4
escaper order 2@ 0 2 0-10 4
escaper order 2@ 0 2 0-19 4
escaper order 2@ 0 2 0-5 4
escaper order 2@ 1 12 0-10 4
#good
escaper order 1@ 0 2 1-10 5
escaper order 3@ 2 4 0-02 4
escaper order 3@ 2 4 0-03 4
escaper order 5@ 16 8 0-08 5
escaper order 6@ 6 11 1-23 5


escaper order 4@ 6 11 2-23 5
#bad
room remove 0 2
company remove company1@gmail
company remove company5@gmail



report day
#good
company remove company5@gmail
escaper order 2@ 8 2 0-10 5
#bad
escaper order 1@ 0 2 0-10 5
escaper order 3@ 0 2 0-10 5
escaper order 1@ 1 12 0-11 5
escaper order 1@ 16 8 0-11 5

escaper remove 4@

report day 
#bad
escaper remove 4@

report best 

#nothing to report
report day

escaper recommend 3@ 6
report day