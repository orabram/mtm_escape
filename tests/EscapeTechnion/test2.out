Day #0: 5 events in total
The order was made by 3@, from the faculty of Civil and Environmental Engineering, who is skilled at 1
	It was made for room 4, of company2@gmail from Electrical Engineering, for the time 2:00
	The room is best for a skill level of 5. 4 people came for the total price of 80
The order was made by 3@, from the faculty of Civil and Environmental Engineering, who is skilled at 1
	It was made for room 4, of company2@gmail from Electrical Engineering, for the time 3:00
	The room is best for a skill level of 5. 4 people came for the total price of 80
The order was made by 2@, from the faculty of Mechanical Engineering, who is skilled at 4
	It was made for room 2, of company1@gmail from Civil and Environmental Engineering, for the time 5:00
	The room is best for a skill level of 4. 4 people came for the total price of 320
The order was made by 5@, from the faculty of Physics, who is skilled at 1
	It was made for room 8, of company5@gmail from Humanities and Arts, for the time 8:00
	The room is best for a skill level of 2. 5 people came for the total price of 300
The order was made by 1@, from the faculty of Civil and Environmental Engineering, who is skilled at 1
	It was made for room 2, of company1@gmail from Civil and Environmental Engineering, for the time 10:00
	The room is best for a skill level of 4. 4 people came for the total price of 240
That is all for day #0!

Day #1: 3 events in total
The order was made by 1@, from the faculty of Civil and Environmental Engineering, who is skilled at 1
	It was made for room 2, of company1@gmail from Civil and Environmental Engineering, for the time 10:00
	The room is best for a skill level of 4. 5 people came for the total price of 300
The order was made by 2@, from the faculty of Mechanical Engineering, who is skilled at 4
	It was made for room 2, of company4@gmail from Physics, for the time 10:00
	The room is best for a skill level of 8. 5 people came for the total price of 120
The order was made by 6@, from the faculty of Humanities and Arts, who is skilled at 7
	It was made for room 11, of company6@gmail from Industrial Engineering and Management, for the time 23:00
	The room is best for a skill level of 9. 5 people came for the total price of 600
That is all for day #1!

The system has 18 faculties and in 2 days, they have made a revenue of 2040
One of the best faculties - Civil and Environmental Engineering, has made a revenue of 860 so far!
One of the best faculties - Industrial Engineering and Management, has made a revenue of 600 so far!
One of the best faculties - Humanities and Arts, has made a revenue of 300 so far!

Day #2: 0 events in total
That is all for day #2!

Day #3: 1 events in total
The order was made by 3@, from the faculty of Civil and Environmental Engineering, who is skilled at 1
	It was made for room 1, of company2@gmail from Electrical Engineering, for the time 8:00
	The room is best for a skill level of 1. 6 people came for the total price of 24
That is all for day #3!

